src/components/Modals/UseAsAliasModal/UseAsAliasModal.container.ts

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux'
+import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { RootState } from 'modules/common/types'
+import { getLoading } from 'modules/ens/selectors'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './UseAsAliasModal.types'
+import UseAsAliasModal from './UseAsAliasModal'
+import { changeProfileRequest, CHANGE_PROFILE_REQUEST } from 'modules/ens/actions'
+
+const mapState = (state: RootState): MapStateProps => ({
+  address: getAddress(state),
+  isLoading: isLoadingType(getLoading(state), CHANGE_PROFILE_REQUEST)
```

Comment:
Given that you're already connecting the Component to the store for isLoading (which is totally correct), what do you think of just getting the current name here? (`yield getName(state)`)
I think it's a bit cleaner to be able to call this modal like this:
`onOpenModal('UseAsAliasModal', { name })`
and also rids the Component that's calling it from that dependency

What do you think?